### ISSUE TITLE
Support Number Pad / Channel selection

### ISSUE THREAD

**Issue by dschmidt** [AShifter/harmony-span#41]
Hey,

it would be nice to be able to select channels directly via the number pad (or even the channel selection with icons).
I understand this is currently not possible because the Roku device harmony-span emulates does not support that.

I assume it's @michaelewers who mentioned in https://www.harmony-remote-forum.de/viewtopic.php?t=16665 he was playing with a Sony device instead. It would be cool, if that approach could be taken further.

Thanks and kudos for this nice project!
I was trying to get remote control for the MagentaTV Box working via the Harmony remote control and found no solution that was working ootb, all needed some tinkering with OpenHAB or ioBroker and seemed way to complicated for my simple use case.

So I took a few roku pieces from this project and some MagentaTV Box remote control pieces from https://github.com/h3llh0und/iobroker.magentatv and merged it together into one project:
https://github.com/dschmidt/harmony-magentatv-bridge

I kept the copyright information in the LICENSE file and added attribution to this and the other project into the README file. I'd be happy to know, this is okay with you - if not, let me know what I can do, like adding more attribution links to copied code or all contributor names of this repo to my LICENSE file.

**Comment by michaelewers** [comment]
Yes, that was me. Unfortunately I've had no luck with Sony TVs so far.
Your fork looks promising.

**Comment by dschmidt** [comment]
Okay, bummer!

Well, apart from the missing number pad support, it does what it aimed to do :-) 
It's been working flawlessly over the whole January, so thanks again for providing your code!
